ci/script.sh

```diff
@@ -32,7 +32,7 @@ case $TARGET in
     ;;
 
   tests)
-    RUST_MIN_STACK=8388608 cargo test --workspace --release --features test-benchmarks,try-runtime
+    RUST_MIN_STACK=8388608 cargo test --workspace --release --features test-benchmarks,try-runtime --exclude runtime-integration-tests
```

Comment:
Do we want this? If we still have issues with the pools tests, let's just disable those and not the whole integration-tests suite 🙏

Reply:
Since we are running the integration tests with its own command `./scripts.sh integration` and those all pass, it seems to me that the `./scripts.sh tests` should only focus on the unit tests then and we can disable the integration test from that run, since they are covered in the other one. 
If we still want to run the integration tests from the unit tests run we need to fix the context setup of the `./scripts.sh tests`, we can do that in a different PR. We are not losing coverage here.